### PR TITLE
translation spanish

### PR DESCRIPTION
**Pull request by dewasha**
translation to spanish

### CODE REVIEW

**Comment by dewasha** [comment]
translation to spanish

File: MainMenu/MainMenuUI/MainMenuUI.es-es.json [hunk 1]
```diff
@@ -0,0 +1,972 @@
...
+      "Identifier": "Settings_Audio",
+      "Text": "AUDIO\r\r\r\n\r\r\n\r\n"
+    },
+    {
+      "Identifier": "Settings_AudioMusic",
+      "Text": "Música"
+    },
+    {
+      "Identifier": "Settings_AudioSFX",
+      "Text": "SFX\r\r\r\n\r\r\n\r\n"
```
Comment (sundhaug92) [inline MainMenu/MainMenuUI/MainMenuUI.es-es.json:305]: Excessive `\r\n`

File: MainMenu/MainMenuUI/MainMenuUI.es-es.json [hunk 2]
```diff
@@ -0,0 +1,972 @@
...
+      "Identifier": "Settings_ClearData",
+      "Text": "BORRAR TODOS LOS DATOS DEL JUGADOR"
+    },
+    {
+      "Identifier": "Settings_OutputLive",
+      "Text": "EXPORTACIÓN DE DATOS AL ORDENADOR"
+    },
+    {
+      "Identifier": "Settings_OutputLiveDesc",
+      "Text": "Esto exportara\r\r\r datos del OBC, AGC y el panel LGC en tiempo real a los archivos dedicados in las carpetas de Exportación (en la carpeta AppData en Reentry). Esto puede usarse para mostrar los datos en un dispositivo externo (como loT).\n\r\r\n\r\n"
```
Comment (sundhaug92) [inline MainMenu/MainMenuUI/MainMenuUI.es-es.json:377]: Excessive `\r\n`, possibly spurious `\r`

File: MainMenu/MainMenuUI/MainMenuUI.es-es.json [hunk 3]
```diff
@@ -0,0 +1,972 @@
...
+      "Identifier": "GraphicsSettings_Title",
+      "Text": "CONFIGURACIONES GRÁFICAS\r\r\n\r\n"
+    },
+    {
+      "Identifier": "GraphicsSettings_ProbeQualityTitle",
+      "Text": "Calidad de reflejo de la Sonda\r\r\n\r\n"
+    },
+    {
+      "Identifier": "GraphicsSettings_ProbeQualityDesc",
+      "Text": "Las sondas de reflejo son usadas para renderizar todo tipo de reflejos. Desde luces de reflejo de plantes mientras estás en la orbita (sondas generales), hasta los reflejos internos de la cabina, vistos en los indicadores, ventanas y sistema reflectivos (sondas criticas). Provocan un gran impacto en tu sistema.\r\r\nSi usas una frecuencia menor de refresco, sugiero desactivar el DESLUMBRAMIENTO DE LA VENTANA.\r\r\n\r\n"
```
Comment (sundhaug92) [inline MainMenu/MainMenuUI/MainMenuUI.es-es.json:429]: Excessive `\r\n`

File: MainMenu/MainMenuUI/MainMenuUI.es-es.json [hunk 4]
```diff
@@ -0,0 +1,972 @@
...
+    },
+    {
+      "Identifier": "GraphicsSettings_ProbeQualityInfo",
+      "Text": "APAGADO: Todas las sondas de reflexión se apagarán\r\r\r\nBAJO: Las sondas críticas se actualizarán en catorce fotogramas, Resolución máxima: 256\r\r\r\nMEDIA: Las sondas críticas se actualizarán en nueve fotogramas, Máx. resolución: 512\r\r\r\nALTA: Las sondas críticas se actualizarán dentro de un marco.\r\r\r\nULTIMATE: Todas las sondas se actualizarán dentro de un marco.\r\r\r\n\r\ r\n\r\n"
```
Comment (sundhaug92) [inline MainMenu/MainMenuUI/MainMenuUI.es-es.json:433]: Excessive `\r\n`, spurious `\ `

File: MainMenu/MainMenuUI/MainMenuUI.es-es.json [hunk 5]
```diff
@@ -0,0 +1,972 @@
...
+    },
+    {
+      "Identifier": "GraphicsSettings_PostProcessingTitle",
+      "Text": "Post Procesado\r\r\n\r\n"
```
Comment (sundhaug92) [inline MainMenu/MainMenuUI/MainMenuUI.es-es.json:437]: Excessive `\r\n`

**Review by sundhaug92 — CHANGES_REQUESTED** [review]
Some changes needed, some spurious/excessive escaped characters

File: MainMenu/MainMenuUI/MainMenuUI.es-es.json [hunk 6]
```diff
@@ -0,0 +1,972 @@
...
+      "Identifier": "ProgramInfo_Apollo",
+      "Text": "El proyecto Apollo es el programa que eventualmente hizo aterrizar hombres en la Luna."
+    },
+    {
+      "Identifier": "ProgramInfo_MCC",
+      "Text": "El Centro de Control del Mercury, es un modo multijugador donde multiples controladores de misión deben apoyar una misión Mercury."
+    },
+    {
+      "Identifier": "ProgramInfo_MOCR",
+      "Text": "El Centro \rde Control de Misiones en Vivo, permite que multiples controladores apoyen una misión Apollo a traves del MOCR."
```
Comment (sundhaug92) [inline MainMenu/MainMenuUI/MainMenuUI.es-es.json:201]: Spurious `\r`?

File: MainMenu/MainMenuUI/MainMenuUI.es-es.json (hunk 3 above)
Comment (sundhaug92) [reply MainMenu/MainMenuUI/MainMenuUI.es-es.json:429]: Spurious `\r` and `\n`

File: MainMenu/MainMenuUI/MainMenuUI.es-es.json [hunk 7]
```diff
@@ -0,0 +1,972 @@
...
+      "Identifier": "RealismSettings_AutoProceedChecklistDesc",
+      "Text": "Si usa la función EJECUTAR (RUN) en las listas de verificación, la guía continuará automáticamente hasta que llegue a un paso que requiera su atención. Si esto está deshabilitado, debe presionar PRO para pasar al siguiente paso."
+    },
+    {
+      "Identifier": "RealismSettings_SimplifiedDockingTitle",
+      "Text": "Combustion y encuentros simplificados"
+    },
+    {
+      "Identifier": "RealismSettings_SimplifiedDockingDesc",
+      "Text": "Habilitar esto hará que los encuentros y las maniobras orbitales sean un poco más simples con algunas correcciones automáticas de guía cuando la lógica detecta errores acumulados o desplazamientos de objetivos. Recomiendo dejar esto activado, especialmente cuando las computadoras de orientación se encuentran en sus primeras etapas. La lógica del juego puede anular esto en lugares críticos".
```
Comment (sundhaug92) [inline MainMenu/MainMenuUI/MainMenuUI.es-es.json:553]: Invalid JSON? . appears outside of `""`

File: MainMenu/MainMenuUI/MainMenuUI.es-es.json [hunk 8]
```diff
@@ -0,0 +1,972 @@
...
+      "Identifier": "RealismSettings_LunarLandingDirectionDesc",
+      "Text": "Habilitar esto orientará automáticamente el módulo lunar para mirar hacia la dirección del lugar de aterrizaje cuando se haga la transición al área de aterrizaje lunar."
+    },
+    {
+      "Identifier": "RealismSettings_MercuryRandomFailuresTitle",
+      "Text": "Fallos Aleatorios: Mercury"
+    },
+    {
+      "Identifier": "RealismSettings_MercuryRandomFailuresDesc",
+      "Text": "Esto habilitará fallos o averias aleatorias en misiones que lo permitan (Free Play o misiones personalizadas). El sistema de fallos aleatorios lanzará un dado de probabilidades una vez cada 2 a 5 minutos (ciclo), donde un fallo aleatorio tiene la posibilidad de ocurrir. A continuación, puede configurar la posibilidad de que ocurra un fallo aleatorio para cada ciclo y qué fallos desea excluir de la lista. Tenga en cuenta que puede activar manualmente un fallo/averia en el juego usando el comando de consola de fallos".
```
Comment (sundhaug92) [inline MainMenu/MainMenuUI/MainMenuUI.es-es.json:585]: . appears after "

File: MainMenu/MainMenuUI/MainMenuUI.es-es.json [hunk 9]
```diff
@@ -0,0 +1,972 @@
...
+      "Identifier": "RealismSettings_GeminiRandomFailuresToggleFailures",
+      "Text": "CAMBIAR AVERIAS"
+    },
+    {
+      "Identifier": "RealismSettings_ApolloRandomFailuresTitle",
+      "Text": "Random Failures: Apollo"
+    },
+    {
+      "Identifier": "RealismSettings_ApolloRandomFailuresDesc",
+      "Text": "Esto habilitará fallos o averias aleatorias en misiones que lo permitan (Free Play o misiones personalizadas). El sistema de fallos aleatorios lanzará un dado de probabilidades una vez cada 2 a 5 minutos (ciclo), donde un fallo aleatorio tiene la posibilidad de ocurrir. A continuación, puede configurar la posibilidad de que ocurra un fallo aleatorio para cada ciclo y qué fallos desea excluir de la lista. Tenga en cuenta que puede activar manualmente un fallo/averia en el juego usando el comando de consola de fallos"."
```
Comment (sundhaug92) [inline MainMenu/MainMenuUI/MainMenuUI.es-es.json:617]: `"."` -> `."`

**Review by sundhaug92 — CHANGES_REQUESTED** [review]

**Pull request merged**